addon/tsconfig.json

```diff
@@ -0,0 +1,44 @@
+{
+  "compilerOptions": {
```

Comment:
You should switch this over to just using `@tsconfig/ember` (install exactly that package!),  now that it's available. Also, let's rename the `types` directory to something unambiguous like `dev-only-types` so it's clear they won't be published!

```jsonc
{
  "extends": "@tsconfig/ember",
  "compilerOptions": {
    "baseUrl": "./src",
    "paths": {
      // No paths, no absolute imports, only node_modules imports allowed
      // But fallback for type-overrides and such
      "*": ["dev-only-types/*"]
    }
}
```